public/examples/internetExplorer.html

```diff
@@ -102,7 +103,7 @@
     </script>
     <script src="../dist/js/iePolyfill.js"></script>
 
-     <script src="https://libraries.unbxdapi.com/search-sdk/v0.1.4/vanillaSearch.js"></script>
+     <script src="../dist/js/vanillaSearch.js"></script>
```

Comment:
Do not add local links in example files as customers will refer to this

Reply:
removed